src/Faker/Provider/Base.php

```diff
@@ -263,7 +263,7 @@ public static function shuffle($arg = '')
      * @param array $array The set to shuffle
      * @return array The shuffled set
      */
-    public static function shuffleArray($array = array())
+    public static function shuffleArray(array $array = array())
```

Comment:
@GrahamCampbell thanks. I've forgot to enable errors; It really throw warning for this case. I have removed this changes.